### PR TITLE
chore: changelog 1.22.1

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Preparing 1.22.1 https://github.com/appium/appium/pull/15999

Branch: https://github.com/appium/appium/tree/releases/1.22

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by KazuCocoa** [comment]
I'd like to wait for https://github.com/appium/appium/issues/16050 (unzip one) for this release in case we should prove a capability to handle zip or temporary makes it `false` (and add caps later) for iOS.

**Comment by KazuCocoa** [comment]
My last comment https://github.com/appium/appium/pull/16002#issuecomment-965951551 was ok

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
+CHANGES IN VERSION 1.22.1 (FROM 1.22.0)
+===================================
+
+Appium 1.22.1 is a patch release
+
+We're going to move to [Appium 2.x](https://github.com/appium/appium/blob/2.0/docs/en/about-appium/1.x-eol.md).
+Appium 1.x will be receiving minor or patch releases only if XCTest gets breaking updates or makor bugs pop up before the EOL.
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:7]: major bugs

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+
+### General
+* Allow unzip with JS implementation when `APPIUM_PREFER_SYSTEM_UNZIP` environment variable is given as `0` or `false` [appium-base-driver#537](https://github.com/appium/appium-base-driver/pull/537)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:10]: Allow->Enforce

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,46 @@
...
+### Android General
+
+* Use bundletool to install .apks packages [appium-adb#578](https://github.com/appium/appium-adb/pull/578)
+* Handle unlock via `locksettings` adb command [appium-adb#577](https://github.com/appium/appium-adb/pull/577)
+* Add a tiny delay after changing IME for Android 12 [appium-adb#581](https://github.com/appium/appium-adb/pull/581)
+
+### Android(UIAutomator2)
+
+* Add capabilities:
+    * `unlockStrategy` to customize the strategy of unlock. Defaults to `locksettings`. The previous behavior is `uiautomator`.
```
Comment (mykola-mokhnach) [inline CHANGELOG.md:22]: Defaults to `locksettings` on Android 8+, where this adb shell command is supported.

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**